components/brave_shields/browser/ad_block_service_browsertest.cc

```diff
@@ -744,3 +745,34 @@ IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, CancelRequestOptionTest) {
   EXPECT_TRUE(as_expected);
   EXPECT_EQ(browser()->profile()->GetPrefs()->GetUint64(kAdsBlocked), 1ULL);
 }
+
+// Load a page with a script which uses a redirect.
+IN_PROC_BROWSER_TEST_F(AdBlockServiceTest,
+    RedirectRulesAreRespected) {
+  UpdateAdBlockInstanceWithRules("js_mock_me.js$redirect=noopjs",
+    "["
```

Comment:
clang-format.
Also you may reformat it as a raw string literal to improve readability:
```
R"([
     { "name": ...
     }
   ]
)";```